packaging/conda/build.sh

```diff
@@ -3,29 +3,21 @@
 # Set-up the shell to behave more like a general-purpose programming language
 set -euo pipefail
 
-# Choose the build preset for macOS/Linux
-if [[ "$(uname)" == "Linux" ]]
-then
-  CMAKE_PRESET="linux-gcc-release"
-else
-  CMAKE_PRESET="macos-clang-release"
-fi
-
 # Configure project
-cmake --fresh --preset $CMAKE_PRESET -DBUILD_JARS=OFF -DTESTING=OFF
+cmake -B conda-build -S . -D BUILD_JARS=OFF -D TESTING=OFF -D CMAKE_BUILD_TYPE=Release -G Ninja
```

Comment:
I would have kept the `build` directory name, to make the change minimal. Or, also `/build/conda/` instead of `/conda-build/` ?

Reply:
ok to `build/conda`

Reply:
Done.